NewForce-Cohort-9/tabloid-full-stack-api-all-stars
# #19 View a Post's Comments

### Description

The functionality to view all comments associated with a post has been added. 

Given an admin has logged in to the app, when they are on a Post Details page, they should be able to click a `View Comments` link. When clicked, a user will navigate to a page listing all post comments from newest to oldest. User should be able to click the post title at the top of the page to return back to the Post Details view.

[Trello Ticket #19 View a Post's Comments](https://trello.com/c/o76RDBCe)

### Type of Change


### Testing Instructions

1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout <js-ViewPostComments>` 
5. If you don't see the most recent version of my files, `git pull origin <js-ViewPostComments>`
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
9. Click the execute with debugger solid green button in purple Visual Studio to launch the api
10. In your terminal, launch the program within the root folder with the command `npm start`
11. In browser, please go to `Login` on the nav bar and log in with this email: `foo@bar.com`
12. This should redirect you back to the home page. 
13. You should  see `Posts` on the nav bar.
14. Select `Posts` to be directed to `/posts` where you should see a list of all posts.
15. Select the `morph front-end markets` post to enter post details for that post
16. Select "View Comments" to view a list of comments that are associated with that post in order from newest to oldest.
17. Click the post title to return to the post details view.

### Checklist:
